src/resources/app-en.json

```diff
@@ -944,13 +944,15 @@
     "abTrailsMobile": "A/B Trails Mobile",
     "flanker": "Simple & Choice Reaction Time Task Builder",
     "gyroscope": "CST Gyroscope",
-    "touch": "CST Touch"
+    "touch": "CST Touch",
+    "unity": "New Unity Task (Beta)"
```

Comment:
While we don't have a name yet I'd avoid "New" and keep it as "Unity Task (Beta)" or (POC)